guides.md

```diff
@@ -800,6 +1030,45 @@ To record or update the voicemail greeting [access the voicemail](#accessing-voi
 - 0 - Mailbox Options
 - 1 - Record your unavailable message
 
+## VPN
+We use pfSense to manage OpenVPN users and gain access to internal resources and also provides internet access over a Canadian IP address.
```

Comment:
```suggestion
We use pfSense to manage OpenVPN users and gain access to internal resources and also provides internet access over a Canadian IP address.

### Using the VPN

To use the VPN you require to have
 - an installed [OpenVPN client](https://openvpn.net/community-downloads/) on your device
 - a configuration file that is generated by pfsense. You will receive this from the member that sets up your account.
 - a username and password. This will be shared with you in Passbolt.
```